tools/systemd-units/char-rathena.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Rathena Char Server
+After=network-online.target remote-fs.target nss-lookup.target
+Wants=network-online.target
```

Comment:
```suggestion
```
I dont see any benefit in defining this dependency (in addition to the startup order).

Reply:
Thats already defined by "After". Is it not?
"Wants" does not give any additional benefit, since even if starting the network fails, it will still try to start the unit.